### DISCUSSION TITLE
Java client for EnvKey

### DISCUSSION THREAD

**Discussion by gnschenker** [envkey/envkey#15] [Q&A] [answered]
We are considering using EnvKey in our Java/SpringBoot microservices. Is there a recommended Java client for EnvKey available? So far I found this: https://github.com/rmannibucau/envkey-client
I wonder if there are others?

**Comment by danenania** [comment, marked as answer]
Hi @gnschenker. The recommended way to use EnvKey with Java is with the envkey-source CLI tool (the library you linked isn't up-to-date or maintained).

Check out the [integration quickstart](https://docs-v2.envkey.com/docs/integration-quickstart) for step-by-step instructions. Let me know if anything isn't clear or you aren't sure how it would adapt to your Java workflow.
